apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,310 @@
+          "Message displayed when an error occurs while updating notification settings.",
+      }),
+    );
+  };
+
+  const handleFormSubmit: SubmitHandler<FormValues> = async (
+    values: FormValues,
+  ) => {
+    const data = formValuesToData(values);
+    await executeMutation(data)
+      .then((result) => {
+        if (result.data?.updateIgnoredNotifications) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage: "Successfully updated settings",
+              id: "ennrdC",
+              description:
+                "Message displayed when applicant updated their notification settings.",
+            }),
+          );
+        } else {
+          handleError();
+        }
+      })
+      .catch(() => {
+        handleError();
+      });
+  };
+
+  const formFields = [
+    {
+      id: "systemMessages",
+      name: "systemMessages",
+      legend: intl.formatMessage({
+        defaultMessage: "System messages",
+        id: "HBmwNb",
+        description: "Title for system messages checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "These notifications are required and include email verification and other important communications.",
+        id: "blvpQr",
+        description: "Subtitle for system messages checklist",
+      }),
+      disabled: true,
+    },
+    {
+      id: "applicationUpdates",
+      name: "applicationUpdates",
+      legend: intl.formatMessage({
+        defaultMessage: "Application updates",
+        id: "v21LPW",
+        description: "Title for application updates checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "Get notified about upcoming application deadlines, status changes, and more.",
+        id: "XzyFuo",
+        description: "Subtitle for application updates checklist",
+      }),
+    },
+    {
+      id: "jobAlerts",
+      name: "jobAlerts",
+      legend: intl.formatMessage({
+        defaultMessage: "Job alerts",
+        id: "fRoNIK",
+        description: "Title for job alerts checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "Get notified when a new job is posted to the platform.",
+        id: "u3CPra",
+        description: "Subtitle for job alerts checklist",
+      }),
+    },
+  ];
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(handleFormSubmit)}>
+        <Card title="" color="white">
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-padding-bottom="base(x1)"
+          >
+            <div
+              data-h2-display="base(flex)"
+              data-h2-gap="base(x1) p-tablet(x2) laptop(x2.5)"
+              data-h2-align-self="base(flex-end)"
+            >
+              <p>{intl.formatMessage(commonMessages.email)}</p>
+              <p>{intl.formatMessage(commonMessages.inApp)}</p>
```

Comment:
I'm not sure what to do here (maybe add `aria-hidden` to these) but since each checkbox has its own label, these feel out of place on an AT.

Reply:
Fixed by [6f0796e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/6f0796e96bdfa9004d300abdae2563e9ce7af4e8)